### PR TITLE
[Form] Add missing invalid_message translations

### PR DESCRIPTION
**Pull request by wouterj**
| Q             | A
| ------------- | ---
| Branch?       | 5.x
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

Some invalid messages were missed in #30931 (probably because of the old age of that PR). This PR adds all missing translation keys and a missing invalid message for the new WeekType.

I've also added the dutch translations (might be up for improvement, but it's a start). We should probably initiate another community initiative to translate these new messages (only a couple languages include the new invalid messages yet).

### CODE REVIEW

File: src/Symfony/Component/Form/Extension/Core/Type/WeekType.php [hunk 1]
```diff
@@ -155,6 +155,11 @@ public function configureOptions(OptionsResolver $resolver)
             },
             'compound' => $compound,
             'choice_translation_domain' => false,
+            'invalid_message' => function (Options $options, $previousValue) {
+                return ($options['legacy_error_messages'] ?? true)
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Form/Extension/Core/Type/WeekType.php]: on one line please

File: src/Symfony/Component/Form/Extension/Core/Type/WeekType.php [hunk 2]
```diff
@@ -155,6 +155,11 @@ public function configureOptions(OptionsResolver $resolver)
             },
             'compound' => $compound,
             'choice_translation_domain' => false,
+            'invalid_message' => function (Options $options, $previousValue) {
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Form/Extension/Core/Type/WeekType.php]: `static` I suppose.

File: src/Symfony/Component/Form/Extension/Core/Type/WeekType.php (hunk 1 above)
Comment (wouterj) [reply src/Symfony/Component/Form/Extension/Core/Type/WeekType.php]: This is copy-pasted from https://github.com/symfony/symfony/pull/30931/files#diff-9c4249df096b31311c1569e2328b1fe154f16d76d2041c48c51e3c01a0add742 Do you want me to update all form types?

Comment (nicolas-grekas) [reply src/Symfony/Component/Form/Extension/Core/Type/WeekType.php]: No need, that'd make merges more difficult.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @wouterj.

**Pull request merged**